### ISSUE TITLE
Calendar CSS are not automatically included when using plain TS version inside Angular

### ISSUE THREAD

**Issue by kamilchlebek** [fullcalendar/fullcalendar-angular#321]
_As discussed here: https://github.com/fullcalendar/fullcalendar/issues/5527 I'm adding new issue with demo._

#### Description

I'm using plain typescript fullcalendar inside my angular application. 
After upgrading to v5 styles are not included automatically. To have styles included I need to use:
```
@import '~@fullcalendar/common/main.css';
@import '~@fullcalendar/list/main.css';
@import '~@fullcalendar/daygrid/main.css';
@import '~@fullcalendar/timegrid/main.css';
```

#### Demo

https://stackblitz.com/edit/angular-ivy-xgkzsq

Uncomment lines from `app.component.scss` to fix the issue.

_To clarify, this is not an issue to me, but it's just different than what I expected after reading migration guide._

**Comment by jlavric** [comment]
Hi!
I don't know if this is still relevant. I have followed the documentation and after upgrade to angular 11 FC just stoped working. I know we have now latest version which is compatible with ng 11 and thank you for that. But still when I do ng build with aot and prod switch somehow the css part of FC was stripped out. I have rebuilded my whole project I did not find a reason why and where this happens. New empty angular 11 project with FC works as it should. 
Anyway my build is getting big it is of 7MB so I went through angular build switches in tsconfig. I have turned **on** **vendor chunks** to **true** which is by  documentation not needed but it does not hurt. In my case the side effect is that suddenly FC CSS is **working**. Without workaraounds.
I hope that this will help anybody :)
